models/reviews.model.js

```diff
@@ -7,13 +7,19 @@ exports.fetchReviewById = (reviewID) => {
       if (result.rows[0] === undefined) {
         return Promise.reject({ status: 404, msg: 'Not found' });
       }
```

Comment:
Rather than get the comment count by making another request to your database, look into doing this through PSQL with things a LEFT JOIN
If we join the comments table on to the review table, we can COUNT the column to get the amount of comments with the original query